econml/dml.py

```diff
@@ -177,22 +182,23 @@ def __init__(self, model, is_Y):
                 self._is_Y = is_Y
 
             def _combine(self, X, W, n_samples, fitting=True):
+                no_x = X is None
+                if no_x:
```

Comment:
When X is None, then XW should just be W, not [1, W]. The extra column of ones, should not be appended to XW.

Reply:
I simplified the logic, which should now handle this correctly.